### ISSUE TITLE
tW and wA no longer load a project to the point where it was last used

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#5551]
1.1.0 (4d06842) 
In previous versions of tC, when the user loaded a project in one of the tools, it opened to where it was last being worked on. Now it loads to the first verse in wA and the first check in tW.
I assume this was broken in one of the issues for refactoring tool loading, but neither of them are in QA yet.

**Comment by da1nerd** [comment]
yeah a lot of code has gone in for the tool loading, but no new functionality has been added yet so they haven't been moved into QA.
I'll investigate this.

**Comment by da1nerd** [comment]
After digging around I discovered this bug was introduced on [this line](https://github.com/unfoldingWord-dev/translationCore/blob/develop/src/js/actions/ContextIdActions.js#L141) on November 14th.

@RoyalSix it looks like you accidentally used `indexOf` instead of `map`.

This line
```js
const contextIdExistInGroups = groupsIndex.indexOf(({id}) => id === contextId.groupId) >= 0;
```

should instead be written like
```js
const contextIdExistInGroups = groupsIndex.map(({id}) => id === contextId.groupId).length >= 0;
```
Just a minor typo :)

Fix is in https://github.com/unfoldingWord-dev/translationCore/pull/5552.

@RoyalSix the comment on your edit in November was "added fix for missing currentContextId from different group". Could you double check that there has been no regression on that?

**Comment by cckozie** [comment]
1.1.0 (0179053) 
[en_ulb_tit_book.zip](https://github.com/unfoldingWord-dev/translationCore/files/2673096/en_ulb_tit_book.zip)

In wA it correctly loads/scrolls to the point it was last used.
In tW it loads to the proper check in the select pane, but it does not scroll the menu to where that check is visible there. The menu remains scrolled to the top.

**Comment by cckozie** [comment]
1.1.0 (b287d3d)
wA now opens to 1:1 and then scrolls visibly to where it was last used. It would be better to work the way it used to, just load to the last used verse rather than display the scrolling. This may confuse the user. I'm checking with @benjore to see if this is acceptable.

tW still opens scrolled to the top check even though the last viewed check is active. I see that this appears to be project (or ???) related. Working on finding the correct sequence of events.
This is a little different, but when the project above is opened in tW, then in wA, and then back in tW, the group menu is really messed up. Things out of sequence (maybe reverses), chapters in the menu, etc. 
This all may be related to #5639
![image](https://user-images.githubusercontent.com/8171759/50610209-b6b9ef00-0e97-11e9-9583-860936992f21.png)
##
![image](https://user-images.githubusercontent.com/8171759/50610229-d18c6380-0e97-11e9-8e27-778b8cdaf625.png)

**Comment by da1nerd** [comment]
This ~may~ will be handled in #4443

**Comment by cckozie** [comment]
1.1.0 (b287d3d)
wA now opens to 1:1 and then scrolls visibly to where it was last used. It would be better to work the way it used to, just load to the last used verse rather than display the scrolling. This may confuse the user. I'm checking with @benjore to see if this is acceptable.

tW still opens to the top check even though the last viewed check is active.

**Comment by cckozie** [comment]
1.1.0 (1034612) 
tW is now scrolling to the last active check, but wA is again always opening to verse 1:1

**Comment by da1nerd** [comment]
This fix has finally been prepare in a PR https://github.com/translationCoreApps/tc-ui-toolkit/issues/164

**Comment by cckozie** [comment]
Fixed in 1.1.1 (f6a4c6b)
